release/models/grpc/openconfig-grpc-services-common.yang

```diff
@@ -0,0 +1,57 @@
+module openconfig-grpc-services-common {
+  yang-version "1";
+  namespace "http://openconfig.net/yang/grpc/common";
+  prefix "oc-grpc-common";
+
+  import openconfig-extensions { prefix oc-ext; }
+
+  organization
+    "OpenConfig working group";
+  contact
+    "www.openconfig.net";
+
+  description
+    "";
+
+  oc-ext:openconfig-version "0.1.0";
+  oc-ext:catalog-organization "openconfig";
+  oc-ext:origin "openconfig";
+
+  revision 2024-03-10 {
+    description
+      "Initial revision.";
+    reference "0.1.0";
+  }
+
+  grouping common-config {
+    description
+      "";
+
+    leaf enable {
```

Comment:
Is there a need for the `enable` leaf for each service?

Reply:
Possibly - my point was rather first to draw out the modular skeleton hierarchies to demonstrate what per service/RPC config/state could expand upon for comment.

Currently, the bucketization of a `GRPC_SERVICE` today is done mostly on a "project" basis (e.g. gNOI) and each project can encapsulate >= 1 gRPC "services" and each can comprise of >= 1 RPCs -> message structs -> fields and so on.  The behaviors of some are more well defined than others (some may carry a specification, some vast comments, some few comments)

So as we break this down, there could be a necessity for granularity at each of these layers.  Enable/Disable of a service currently could be accomplished by removing the entry from the leaf-list so there would have to be some discussion on what this means, if it is of value, etc...

But at this layer could be various other branches to represent the above mentioned breakdown where state or config override behaviors, etc... could exist

This common model is meant to be set of nodes that would apply to every gRPC service (enable was an easy example is all....)

Reply:
> Enable/Disable of a service currently could be accomplished by removing the entry from the leaf-list so there would have to be some discussion on what this means, if it is of value, etc...
Yep, that's pretty much my point. The existing leaf-list already accomplishes that, and the explicit enable appears to be redundant. 

Now I see what you mean, thanks.